Unity-project/bad code/TouchJoystickRotation.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TouchJoystickRotation : MonoBehaviour
+{
+	public Joystick joystick;
+	public GameObject Object;
+	public GameObject ChildObject;
+	public SpriteRenderer SRObject;
+	public bool IsTrigger;
+	public float x;
+	Vector2 GameobjectRotation;
+	private float GameobjectRotation2;
+	private float GameobjectRotation3;
+
+	public bool FacingRight = true;
+
+	void Update()
+	{
+		//Gets the input from the jostick
+		//GameobjectRotation = new Vector2(joystick.Horizontal, joystick.Vertical);
+
+		//GameobjectRotation3 = GameobjectRotation.x;
+
+		//if (joystick.Horizontal > 0)
+		//{
+		//	//Rotates the object if the player is facing right
+		//	GameobjectRotation2 = GameobjectRotation.x + GameobjectRotation.y*90;
+		//	Object.transform.rotation = Quaternion.Euler(0f, 0f, GameobjectRotation2);
+		//}
+		//else
+		//{
+		//	//Rotates the object if the player is facing left
+		//	GameobjectRotation2 = GameobjectRotation.x + GameobjectRotation.y * -90;
+		//	Object.transform.rotation = Quaternion.Euler(0f, 180f, -GameobjectRotation2);
+		//}
+		//if (GameobjectRotation3 < 0 && FacingRight)
+		//{
+		//	// Executes the void: Flip()
+		//	//Flip();
+		//}
+		//else if (GameobjectRotation3 > 0 && !FacingRight)
+		//{
+		//	// Executes the void: Flip()
+		//	//Flip();
+		//}

```

Comment:
Commented code can appears in development branches but should not be landed to master/main branch